### PR TITLE
Upgrade to Play 2.9.x

### PR DESCRIPTION
**Pull request by bion**
### Description

Upgrade to Play 2.9.x

Details are in the issue itself

closes https://github.com/civiform/civiform/issues/5647

Rebasing to merge previous commits to make it easier to keep in sync with main

### These are previous commit messages

- Play HTTP Updates
- Make questions index synchronous
- Make the other question service actions sync
- Update logback version and config
- The Play 2.9 TestServer now uses a dynamically assigned port. Setting it back to the expected fixed port number to get tests working again.
- Play 2.9 now enforces a minimum length for the play.http.secret.key.
- Updating the frozen test cookie.
- Renaming httpContext variables
- Fixing deprecated method call
- Set play to v2.9.2

### CODE REVIEW

File: test-support/prod-simulator-compose.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ services:
     ports:
       - 8888:9000
     environment:
-      - SECRET_KEY=notarealsecret
+      - SECRET_KEY=K5SgucxBYC3xJwNcGWZV1Y7uASmrc+SqftbcTROH0s8=
```
Comment (dkatzz) [inline test-support/prod-simulator-compose.yml:22]: **nitpick:** Maybe add a comment above that this is a randomly generated secret that isn't real but complies with play's secret length

File: browser-test/src/support/config.ts [hunk 2]
```diff
@@ -11,5 +11,5 @@ export const {
...
...swb1BwcFA3NTlUMzR1NXAvTndMNk5qS2ovL0dyZDhaUUN5UU1FcUhvbXJsTEV0VXJPYnFZRlp4dXlOck8wb3Boayt2eFhYY3dzWis4YUZxeHpLWnkvdlBtcnIvd3FVT21SRjhPNm1SZzBqQ2hZRjB4ZFdBa0lQNGlnNUhkZW5tZlptSHcyYmg2N2JVWnB1MGlMREFTQmdGbHhEaDdDamdHRXNGZnRGYmVSRWtoVjFUdzZacWxDZTM3MFR3dXg2NmNmSWp2c092NHN4Q0FMVG4wVGVxbUR4K2tpM29ibWQ1TFNyLzhhb1RBY2kzTGFoOERLczQrVWxKUHpKOGZLVHZkTzFXS0kwSjZ0MjdBbGx2T2N1ZkxtRVJXM0Y2dnpVU0l5bTVMa1ZSS01hUlFWOG9Da1JESlQrV09xQk1qTDBJdXJ3SUFrQ3UyZCtqUmxTZklqTnBISWVvcDFrUTNEa1BMejM0QVNvNEVNWml1VUhER3lWZkZhWW5aK2YyVWZESkxWeWs2ZmhRNjBZTWlXRG5OMUtHSGlON3UzcVIzZ3grSVZ3OWtONGVQMm5SbmRuTWluQnZnem9qUlE3VXlFcE55eHUvQnBDQTlBRXlLcENpTkxudmJvTmh4UWw2d0MxSmN1Z0NHK1lwRFkycEhOcTl3UXNRZktRRDQwdlo5eGlPRXhHV0tzdkFQTGZ6dllwVFhFMTFjRkdJVTJ4cUgrZW9yelNGMjFqa2laSVlqQmM0N3NpS25HVWordHlVbUh3K0pLdnlqUDhiajB2WUhiN0kzRUJkbERhaTVHTnRNK3BDU0l1Zkg0Mkg1OEtOSkRiMWdKdzRocXl4QzBqL0ZMQmtibUR6SXNwTW16WEY4U3lDT0FvV0o4bjZVZFpyOG52MjVleUVDMDR5VlF1SlVBR3BwZzdnYktqdXd0VnBmN0tlNXowaExiMWpyT05yTFFBQWFSckYzVTZRSm0vVEE9PSIsImNzcmZUb2tlbiI6Ijc2ZWNlNWZlZTRkNzgwZDEzZTZjZmJiODAzMWI3YTFmMWZjODM2Y2YtMTcwNTcxOTg2NjUzNy01MDJjNWZiZTlmNmFhMmRhYzhhMGE1NDQifSwibmJmIjoxNzA1NzE5ODY2LCJpYXQiOjE3MDU3MTk4NjZ9.WYG9_nsRo4Bjev7IG3rK4JJJfWWBJGhkT1wZDecKmVs',
```
Comment (dkatzz) [inline browser-test/src/support/config.ts:14]: **question:** What's the impact of changing this?

**Review by dkatzz — APPROVED** [review]
Thank you so much for doing this!!! Just one question and one nit, but overall LGTM.

File: browser-test/src/support/config.ts (hunk 2 above)
Comment (gwendolyngoetz) [reply browser-test/src/support/config.ts:14]: Tom added it for some tests. It is changing because the secret key changed.

Comment (dkatzz) [reply browser-test/src/support/config.ts:14]: Oh right! Makes sense.

**Comment by gwendolyngoetz** [comment]
@nb1701 We'll need to update `javax.persistence` to `jakatra.persistence` when we move to play-ebeans 7.x, but it's not needed right now.

**Comment by gwendolyngoetz** [comment]
> @gwendolyngoetz Are we just waiting on a green light to merge this once we think everyone is updated to the longer secret?

Yep. If we want to cut an out of band release to run on staging we could do that too for more testing.

I'm trying to rebase it one or twice a week to make sure it stays in sync.

**Review by dkatzz — APPROVED** [review]
Yay!!

**Pull request merged**